src/pages/edit-profile/EditProfile.styles.ts

```diff
@@ -20,14 +25,24 @@ export const styles = {
     padding: '10px'
   },
   sidebarMenu: {
-    width: '30%'
+    width: '30%',
+    pr: { lg: '96px' }
   },
   mainContainer: {
     display: 'flex',
-    justifyContent: 'space-between'
+    justifyContent: 'space-between',
+    '& .MuiButtonBase-root:hover': {
+      backgroundColor: palette.basic.grey,
+      '& .MuiTypography-body1': {
+        color: palette.basic.black,
+        fontWeight: '600'
+      },
+      '& .MuiListItemIcon-root svg': {
+        color: palette.basic.black
+      }
```

Comment:
These selectors look a bit confusing. `EditProfile` component uses `SidebarMenu` component, which is used only once in our app. Please consider adding this styles to button directly or at least move it from `mainContainer` to `sidebarMenu` style and make more speciffic selector to make style more modular